README.md

```diff
@@ -1,2 +1,9 @@
 # git_host_apis
-git host apis
+raw git host apis
```

Comment:
Would it make sense to mention the dependency to abapGit as prerequisite?
BTW, does anyone use APACK for this purpose these days?

Reply:
in this code there is no runtime dependency to abapgit